src/main/java/com/esewa/usermanagement/configuration/EmailConfigProperties.java

```diff
@@ -0,0 +1,17 @@
+package com.esewa.usermanagement.configuration;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Getter
+@Setter
+@Configuration
+@ConfigurationProperties(prefix = "email")
+public class EmailConfigProperties {
```

Comment:
Can you please create a table to hold these templates?

Reply:
Done